run_tests_examples_and_docs.py

```diff
@@ -0,0 +1,211 @@
+#
```

Comment:
Should this whole module be integrated into the pytest framework?

Reply:
perhaps, though I view testing the examples and docs as separate from pytest

Reply:
and I don't know how to put this in the pytest framework

Reply:
How so?

Reply:
sorry.... I didn't see the "how to put this in the pytest framework" comment before I sent 'how so?'!

Erm, this could be a good reason not to then... but can't you use a pytest fixture to tell it where the docs/notebooks are?